### PR TITLE
Docs - Read Only Users

### PR DESCRIPTION
**Pull request by odscjames**
https://github.com/open-contracting/kingfisher-views/issues/28

### CODE REVIEW

**Comment by odscjames** [comment]
[Make it easier to create new read-only users for the hosted Kingfisher database](https://trello.com/c/BE9JOi2Z/133-make-it-easier-to-create-new-read-only-users-for-the-hosted-kingfisher-database)

**Comment by odscjames** [comment]
@jpmckinney Is this process clear? The idea is that anyone with root access to the server can very quickly add read only users on request.

File: docs/users.rst [hunk 1]
```diff
@@ -6,18 +6,30 @@ View supports the creation of read only users that have access to all views and
...
+    # su postgres
+    $ psql template1
 
 .. code-block:: sql
 
     CREATE USER testreadonly with PASSWORD 'k1ngf1sher' NOCREATEDB NOSUPERUSER NOCREATEROLE;
 
-Then go to the database, and the ``view_meta`` schema, and the ``read_only_user`` table.
-Put the new username in as a entry in that table.
+Then go to the normal views database with your usual connection settings. Put the new username in as an entry in the ``read_only_user`` table in the ``view_meta`` schema.
```
Comment (jpmckinney) [inline docs/users.rst]: What are "your usual connection settings"?

File: docs/users.rst [hunk 2]
```diff
@@ -6,18 +6,30 @@ View supports the creation of read only users that have access to all views and
...
+On a Ubuntu Linux server, after connecting as root this would probably look like:
+
+.. code-block:: bash
+
+    # su postgres
+    $ psql template1
 
 .. code-block:: sql
 
     CREATE USER testreadonly with PASSWORD 'k1ngf1sher' NOCREATEDB NOSUPERUSER NOCREATEROLE;
```
Comment (jpmckinney) [inline docs/users.rst]: It'd be good to recommend/link to a password generator, to have stronger passwords.

**Comment by odscjames** [comment]
Hows that? @jpmckinney

File: docs/users.rst [hunk 3]
```diff
@@ -15,11 +15,13 @@ On a Ubuntu Linux server, after connecting as root this would probably look like
...
 
+Generate a strong password using `passwordsgenerator.net <https://passwordsgenerator.net/>`__ or similiar.
+
 .. code-block:: sql
 
-    CREATE USER testreadonly with PASSWORD 'k1ngf1sher' NOCREATEDB NOSUPERUSER NOCREATEROLE;
+    CREATE USER testreadonly with PASSWORD 'insert-strong-password-here' NOCREATEDB NOSUPERUSER NOCREATEROLE;
 
-Then go to the normal views database with your usual connection settings. Put the new username in as an entry in the ``read_only_user`` table in the ``view_meta`` schema.
+Then go to the normal views database, using the normal username, password and other settings you use to access that. Put the new username in as an entry in the ``read_only_user`` table in the ``view_meta`` schema.
```
Comment (jpmckinney) [inline docs/users.rst]: I'm still not clear on what this normal user is. I don't see any clear instruction on creating/choosing that user on the installation or configuration pages.

Comment (odscjames) [reply docs/users.rst]: https://kingfisher-views.readthedocs.io/en/latest/config.html says "PostgreSQL. Configure the database connection settings - this should point to the same database Kingfisher Process uses:"

How about "Then connect to the normal Kingfisher Process and Views database, using the normal username, password and other settings you use to access that as defined in --LINK TO CONFIGURATION--".

For our hosted kingfisher instance, I could get more exact than that, but if we want to keep this documentation generic, I'm not sure what else to suggest.

**Comment by jpmckinney** [comment]
@odscjames I rewrote the page, as feedback would be too lengthy. Do you see how it is clearer?

Feel free to merge.

**Pull request merged**